### PR TITLE
feat: reverse timeline filter + manage event for timeline

### PR DESCRIPTION
**Pull request by sguiheux**
1. Description
1. Related issues
1. About tests
1. Mentions

@ovh/cds

### CODE REVIEW

File: engine/api/timeline.go [hunk 1]
```diff
@@ -18,27 +19,102 @@ func (api *API) getTimelineHandler() Handler {
...
+		projectFound := false
+		workflowFound := false
+		insert := false
+	projLoop:
+		for _, pf := range filter.Projects {
+			if pf.Key == j.Key {
+				projectFound = true
+				for _, wName := range pf.WorkflowNames {
+					if wName == j.WorkflowName {
+						workflowFound = true
```
Comment (golangci) [inline engine/api/timeline.go]: ineffectual assignment to `workflowFound`

Comment (bnjjj) [reply engine/api/timeline.go]: he's right

File: ui/src/assets/i18n/fr.json [hunk 2]
```diff
@@ -643,7 +643,8 @@
   "timeline_no": "Le fil d'actualité est vide",
   "timeline_title": "Fil d'actualité",
   "timeline_filter_all_project": "Tous les projets",
-  "timeline_filter_project": "Recevoir des évènements sur :",
+  "timeline_filter_mute": "Masquer les évènements de ce workflow",
```
Comment (bnjjj) [inline ui/src/assets/i18n/fr.json]: s/évènement/événement everywhere

Comment (sguiheux) [reply ui/src/assets/i18n/fr.json]: https://www.lalanguefrancaise.com/orthographe/evenement-ou-evenement-orthographe/
La réforme de 1990 me donne raison :)

Comment (bnjjj) [reply ui/src/assets/i18n/fr.json]: cc @antleblanc

**Pull request merged**